engine/runtime/src/main/scala/org/enso/compiler/phase/ImportResolver.scala

```diff
@@ -109,6 +109,13 @@ class ImportResolver(compiler: Compiler) {
                 val converted = bindings
                   .toConcrete(compiler.packageRepository.getModuleMap)
                   .map { concreteBindings =>
+                    compiler.context.updateModule(
```

Comment:
So this is the place where `BindingsMap` is set into the module from the cache retrieved by `compiler.importExportBindings(current)`. Nice.